source/includes/steps-fle-convert-to-a-remote-master-key-azure.yaml

```diff
@@ -0,0 +1,235 @@
+
+  Provide your client with the following information to access the master key:
+
+  .. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Field
+     - Required
+     - Description
+
+   * - keyName
+     - Yes
+     - Name of the master key
+
+   * - keyVersion
+     - No
+     - Version of the master key
+
+   * - keyVaultEndpoint
+     - Yes
+     - URL of the key vault. E.g. myVaultName.vault.azure.net
+
+  Once you have the required information, run the following code to
+  generate the new data encryption key:
+
+  .. tabs-drivers::
+
+     .. tab::
+        :tabid: java-sync
+
+        .. code-block:: Java
+
+           // TODO: update for Azure
+           ClientEncryption clientEncryption = ClientEncryptions.create(ClientEncryptionSettings.builder()
+               .keyVaultMongoClientSettings(MongoClientSettings.builder()
+                   .applyConnectionString(new ConnectionString("mongodb://localhost:27017"))
+                   .build())
+               .keyVaultNamespace(keyVaultNamespace)
+               .kmsProviders(kmsProviders)
+               .build());
+
+           BsonString masterKeyRegion = new BsonString("<Master Key AWS Region>"); // e.g. "us-east-2"
+           BsonString masterKeyArn = new BsonString("<Master Key ARN>"); // e.g. "arn:aws:kms:us-east-2:111122223333:alias/test-key"
+           DataKeyOptions dataKeyOptions = new DataKeyOptions().masterKey(
+               new BsonDocument()
+                   .append("region", masterKeyRegion)
+                   .append("key", masterKeyArn));
+
+           BsonBinary dataKeyId = clientEncryption.createDataKey("aws", dataKeyOptions);
+           String base64DataKeyId = Base64.getEncoder().encodeToString(dataKeyId.getData());
+
+           System.out.println("DataKeyId [base64]: " + base64DataKeyId);
+
+     .. tab::
+        :tabid: nodejs
+
+        .. code-block:: javascript
+
+           // TODO: update for Azure
+           const encryption = new ClientEncryption(client, {
+               keyVaultNamespace,
+               kmsProviders
+           });
+           const key = await encryption.createDataKey('aws', {
+              masterKey: {
+                key: '<Master Key ARN>', // e.g. 'arn:aws:kms:us-east-2:111122223333:alias/test-key'
+              }
+           });
+
+           const base64DataKeyId = key.toString('base64');
+           console.log('DataKeyId [base64]: ', base64DataKeyId);
+
+     .. tab::
+        :tabid: python
+
+        In ``app.py``:
+
+        .. code-block:: python
+
+           master_key = {
+             "keyName": "<Azure key name>",
+             # "keyVersion": "<Azure key vault endpoint>",
+             "keyVaultEndpoint": "<Azure key version>"
+           }
+
+        .. note::
+
+           Make sure you are using `pymongocrypt <https://pypi.org/project/pymongocrypt/>`__
+           version 1.0.2 or later in your application's environment.
```

Comment:
This is incorrect. `pymongocrypt-1.1.0` will be the first version of this library to support Azure. At the moment, we plan to release `1.1.0b0` this week and `1.1.0` when CSFLE is expected to GA.
I think to keep things simple, we should just mention that users need `pymongocrypt-1.1` or later, e.g.:
```
To use the Azure Key Vault, you must use `pymongocrypt <https://pypi.org/project/pymongocrypt/>`__
 version 1.1 or later in your application's environment.
```
The patch version should not be specified here because it can change without warning and is liable to become a maintenance burden.

Reply:
Can we also add a note in the corresponding location for AWS that mentions that users must be using `pymongocrypt` 1.0 or later in their application environment? I checked and there doesn't seem to be one currently.

Reply:
Sounds good, thanks for letting me know.